peachjam/views/judgment.py

```diff
@@ -14,7 +14,9 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         context["court_classes"] = CourtClass.objects.prefetch_related("courts")
-        context["recent_judgments"] = Judgment.objects.order_by("-date")[:30]
+        context["recent_judgments"] = Judgment.objects.exclude(
```

Comment:
Hmm, we need to do the same in all the `HomePageView` classes -- also search for `recent_*` and ensure those are corrected (mostly in homepage views I think).